JitsiConference.js

```diff
@@ -1377,6 +1382,37 @@ JitsiConference.prototype.onIncomingCall
 };
 
 /**
+ * Sets the BridgeChannel.
+ */
+JitsiConference.prototype._setBridgeChannel = function() {
+    const jingleSession = this.jvbJingleSession;
```

Comment:
Do you mean checking if `jingleSession.bridgeWebSocketUrl` is not set? It's already checked in line 1410 ;)